photutils/psf/photometry.py

```diff
@@ -274,10 +278,17 @@ def do_photometry(self, image, positions=None):
                                   sources['ycentroid'],
                                   sources['aperture_flux']])
 
+        self._pars_to_set, self._pars_to_output = self._define_fit_param_names()
```

Comment:
@eteq I went for option 1 because I think it's a bit more intuitive for anyone else looking at the code. Thank you!